tests/test_ignore.py

```diff
@@ -0,0 +1,134 @@
+import pytest
+
+from vulture import core
+
+from . import check, skip_if_not_has_async
+
+
+@pytest.fixture
```

Comment:
Can't we just use the `examine` function without the pytest.fixture? The new function can still be called `check_ignore`.